content/backstage/plugins/bugsnag.md

```diff
@@ -0,0 +1,86 @@
+---
+humanName: Bugsnag
+heading: 'Backstage Bugsnag Plugin'
+lead: 'Monitor errors in Bugsnag for your service in Backstage'
+attribution:
+  text: Roadie
+  href: https://roadie.io
+
+seo:
+  title: 'Backstage Bugsnag Plugin | Roadie'
+  description: |
+    Monitor errors in Bugsnag for your service in Backstage. 
+
+logoImage: '../../assets/logos/bugsnag/bugsnag-logo.png'
+coverImage: '../../assets/backstage/plugins/bugsnag/backstage-bugsnag-plugin.png'
+coverImageAlt: |
+  Service errors overview inside a Bugsnag plugin.
+
+gettingStarted:
+  - intro: Install the plugin into Backstage.
+    language: bash
+    code: |
+      // packages/app
+      'yarn add @roadiehq/backstage-plugin-bugsnag'
+  
+  - intro: Add proxy configurations
+    language: 'yaml'
+    code: |
+      // app-config.yaml
+      proxy:
+        # ...
+
+        '/bugsnag/api':
+          target: 'https://api.bugsnag.com'
+          headers:
+            Authorization: 'token ${BUGSNAG_PERSONAL_TOKEN}'
+            X-version: '2'    
+
+  - intro: Add plugin API to your Backstage instance.
+    language: typescript
+    code: |
+      // packages/app/src/components/catalog/EntityPage.tsx
+      import { EntityBugsnagErrorsOverviewTable } from '@roadiehq/backstage-plugin-bugsnag';
+
+      // ...
+
+      const serviceEntityPage = (
+        <EntityLayout>
+          ...
+          <EntityLayout.Route path="/bugsnag" title="Bugsnag">
+            <EntityBugsnagErrorsOverviewTable />
+          </EntityLayout.Route>
+          ...
+        </EntityLayout>
+      )
+
+  - intro: Run the backstage app with the following command and navigate to the services tab.
+    code: |
+      yarn start
+
+---
+
+## How to use Bugsnag plugin in Backstage:
+
+1.  Add an annotation to the yaml config file of a component:
```

Comment:
I'd suggest dropping the numbered list here. It's a bit weird how the line with the number on it is indented and the rest are not. There's only two steps so simply saying "Firstly, add an annotation..." will be ok.